config/config.yaml

```diff
@@ -30,12 +30,13 @@ service:
     SECRET_KEY: aardvark
     EMAIL_TOKEN_SALT: aardvark
     PARTY_SCHEMA: ras_party/schemas/party_schema.json
+    PUBLIC_EMAIL_VERIFICATION_URL: http://ons.survey.gov/bres                   # Do not add trailing slash e.g. http://x.y/parent/child/ BAD http://x.y/parent/child GOOD
```

Comment:
Not a fan of using defaults like this. If Jenkins is configured wrong the user is going to start getting them without us knowing, can't be good